pkg/infra/proposer_test.go

```diff
@@ -55,4 +58,32 @@ var _ = Describe("Proposer", func() {
 			Expect(err).Should(MatchError(ContainSubstring("error connecting to invalid_addr")))
 		})
 	})
+
+	Context("Tape should do less for prepare and summary endorsement process", func() {
+		Measure("it should do endorsement efficiently for 8 peers", func(b Benchmarker) {
+			concurrent := 8
+			processed = make(chan *infra.Elements, 10)
+			done = make(chan struct{})
+			signeds := make([]chan *infra.Elements, concurrent)
+			for i := 0; i < concurrent; i++ {
+				signeds[i] = make(chan *infra.Elements, 10)
+				mockpeer, mockpeeraddr := infra.StartMockPeer()
+				infra.StartProposer(signeds[i], processed, done, nil, concurrent, mockpeeraddr)
+				defer mockpeer.Stop()
+			}
+			runtime := b.Time("runtime", func() {
+				data := &infra.Elements{SignedProp: &peer.SignedProposal{}}
+				for _, s := range signeds {
+					s <- data
+				}
+				<-processed
```

Comment:
technically, consuming `processed` could be done in a separate goroutine. But it wouldn't make much difference for this test case.